include/openthread/platform/radio.h

```diff
@@ -489,6 +489,16 @@ void otPlatRadioSetMacKey(otInstance *    aInstance,
  */
 void otPlatRadioSetMacFrameCounter(otInstance *aInstance, uint32_t aMacFrameCounter);
 
+/**
+ * Get the current estimated time (64bits width) of the radio chip.
+ *
+ * @param[in]   aInstance    A pointer to an OpenThread instance.
+ *
+ * @returns The current time in microseconds.
```

Comment:
Document the special value when the RCP doesn't support or out of sync with Host?